students/Matt_Hudgins/Lesson03/slicing_lab.py

```diff
@@ -0,0 +1,91 @@
+'''
+    File Name: slicing_lab.py
+    Author: Matt Hudgins
+    Date created: 4/21/18
+    Date last modified: 4/21/18
+    Python Version 3.6.4
+'''
+# The following sequences will be used for the slicing lab exercise
+data = [3, 1, 4, 1, 5, 9, 2, 6, 5, 3, 5]
+string = 'The first ten digits of pi!'
+
+
+def exchange_first_last(seq):
+    """ This function when called will exchange the first and last
+    items of the sequence"""
+    return(seq[-1:] + seq[1:-1] + seq[0:1])
```

Comment:
The last slice here can be `seq[:1]` since it starts at 0 index by default if you leave out the first item of the slice